api/src/main/java/org/datacleaner/descriptors/DescriptorProvider.java

```diff
@@ -46,12 +46,6 @@
 public interface DescriptorProvider {
 
     /**
-     * @return a collection of {@link DescriptorProviderStatus} describing
-     *         current status.
-     */
-    public Collection<DescriptorProviderStatus> getStatus();
-
-    /**
```

Comment:
@LosD @arjansh This method was relative new, we used it only for remote components. This is reason, why I could delete it. But if you wish, I will return it back with inotation.